README.md

```diff
@@ -38,8 +38,7 @@ First, download a pretrained model from the YOLOv6 [release](https://github.com/
 Second, run inference with `tools/infer.py`
 
 ```shell
-python tools/infer.py --weights yolov6s.pt --source img.jpg / imgdir
-                                yolov6n.pt
+python tools/infer.py --weights <yolov6s-or-yolov6n> --source img.jpg / imgdir --wandb_project <wandb-project-name> --wandb_entity <wandb-entity>
```

Comment:
might be too agressive to put wandb in the main inference example code, or if you want to keep this command then add a some text before or after mentioning that the wandb_* arguments are optional and will turn on logging to wandb so you can save and visualise your predictions.